schema/gene.js

```diff
@@ -28,7 +28,7 @@ const SUBJECT_MATTER_REGEX = new RegExp(SUBJECT_MATTER_MATCHES.join("|"), "i")
 const GeneType = new GraphQLObjectType({
   name: "Gene",
   interfaces: [NodeInterface],
-  isTypeOf: obj => _.has(obj, "family") && _.has(obj, "browseable"),
+  isTypeOf: obj => _.has(obj, "browseable") && (_.has(obj, "published") || _.has(obj, "family")),
```

Comment:
Why is this change necessary?

Reply:
The short version of the gene here does not [include the family](https://github.com/artsy/metaphysics/blob/genes_cinnection/test/fixtures/gravity/follow_genes.json#L1) but I think in the other place it's used in the home page, it always includes family

Reply:
Aha, gotcha. I hope the `browseable` and `published` aren’t too generic that they could potentially exist in other payloads, though.

In either case, I don’t think it’s really necessary to do an `or`, just choose a better alternative and use that for all cases.